sessionctx/variable/sysvar.go

```diff
@@ -965,6 +966,8 @@ const (
 	NetWriteTimeout = "net_write_timeout"
 	// ThreadPoolSize is the name of 'thread_pool_size' variable.
 	ThreadPoolSize = "thread_pool_size"
+	// WindowingUseHighPrecision is the name of 'windowing_use_high_precision' system variable.
+	WindowingUseHighPrecision = "windowing_use_high_precision"
```

Comment:
Could we remove this flag...? It seems no one is willing to set it to `OFF`. What're your opinions? @SunRunAway

Reply:
It may have different results in MySQL 8. I prefer keeping it.

```mysql
mysql> set @@windowing_use_high_precision=1;
Query OK, 0 rows affected (0.00 sec)

mysql> select a, sum(a) over(ROWS BETWEEN current row AND 1 FOLLOWING) from t ;
+-----------+-------------------------------------------------------+
| a         | sum(a) over(ROWS BETWEEN current row AND 1 FOLLOWING) |
+-----------+-------------------------------------------------------+
| 100000000 |                                        100000000.0001 |
|    0.0001 |                                0.00009999999747378752 |
+-----------+-------------------------------------------------------+
2 rows in set (0.00 sec)

mysql> set @@windowing_use_high_precision=0;
Query OK, 0 rows affected (0.00 sec)

mysql> select a, sum(a) over(ROWS BETWEEN current row AND 1 FOLLOWING) from t ;
+-----------+-------------------------------------------------------+
| a         | sum(a) over(ROWS BETWEEN current row AND 1 FOLLOWING) |
+-----------+-------------------------------------------------------+
| 100000000 |                                        100000000.0001 |
|    0.0001 |                                0.00010000169277191162 |
+-----------+-------------------------------------------------------+
2 rows in set (0.00 sec)
```

Reply:
Should I keeping it or not？